src/main/resources/data/mappingProfiles/defaultMappingProfiles.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "25d81cbe-9686-11ea-bb37-0242ac130002",
+  "name": "Default instance mapping profile",
+  "description": "Default mapping profile for the inventory instance record",
+  "recordTypes": [
+    "INSTANCE"
+  ],
+  "outputFormat": "MARC",
+  "userInfo": {
+    "firstName": "DIKU",
```

Comment:
Same here, let's call this System process or something, there won't be a diku_admin in all environments